content/applications/services/helpdesk/overview/sla.rst

```diff
@@ -1,47 +1,192 @@
+on a team. Then, click the :guilabel:`SLA Policies` smart button at the top of the team's settings
+page, and click :guilabel:`Create`.
+
+On the blank :abbr:`SLA (Service Level Agreement)` policy form, enter a :guilabel:`Title` and a
+:guilabel:`Description` for the new policy, and proceed to fill out the form using the steps below.
+
+Define the criteria for an SLA policy
+-------------------------------------
+
+The :guilabel:`Criteria` section is used to identify which tickets this policy is applied to.
+
+Fill out the following fields to adjust the selection criteria:
+
+.. note::
+   Unless otherwise indicated, multiple selections can be made for each field.
+
+- :guilabel:`Helpdesk Team`: a policy can only be applied to one team. *This field is required.*
+- :guilabel:`Minimum Priority`: the priority level for a ticket is identified by selecting one, two,
+  or three of the :guilabel:`⭐ (star)` icons, representing the priority level on the Kanban card or
+  on the ticket itself. The :abbr:`SLA (Service Level Agreement)` is **only** applied after the
+  priority level has been updated on the ticket to match the :abbr:`SLA (Service Level Agreement)`
+  criteria. If no selection is made in this field, this policy applies to all priority levels.
+- :guilabel:`Types`: ticket types can be helpful when indicating when a ticket is a customer
+  question, that can be solved with a quick response, or an issue that may require additional
+  investigation. Multiple ticket types can be selected for this field. If no selection is made, this
+  policy applies too all ticket types.
+- :guilabel:`Tags`: tags are used to indicate what the ticket is about. Multiple tags can be applied
+  to a single ticket.
+- :guilabel:`Customers`: individual contacts or companies may be selected in this field.
+- :guilabel:`Sales Order Items`: this field is available only if a team has the *Timesheets* app
+  enabled. This allows the ticket to link directly to a specific line on a sales order, which must
+  be indicated on the ticket in the :guilabel:`Sales Order Items` field.
+
+Establish a target for an SLA policy
+------------------------------------
+
+A *target* is the stage a ticket needs to reach, and the time allotted to reach that stage, in order
+to satisfy the :abbr:`SLA (Service Level Agreement)` policy. Any stage assigned to a team may be
+selected for the :guilabel:`Reach Stage` field.
+
+Time spent in stages selected in the :guilabel:`Excluding Stages` field are **not** included in the
+calculation of the :abbr:`SLA (Service Level Agreement)` deadline.
+
+.. example::
+   An :abbr:`SLA (Service Level Agreement)` titled `8 Hours to Close` tracks the working time before
+   a ticket is completed, and would have `Solved` as the :guilabel:`Reach Stage`. Simultaneously, an
+   :abbr:`SLA (Service Level Agreement)` titled `2 Days to Start` tracks the working time before
+   work on a ticket has begun, and would have `In Progress` as the :guilabel:`Reach Stage`.
+
```

Comment:
Because this is immediately after the Exluding Stage explanation, you could include an example where this applies, or move this example above the Excluding Stages field.